client/my-sites/site-settings/site-icon-setting/index.jsx

```diff
@@ -205,8 +206,14 @@ class SiteIconSetting extends Component {
 					<InfoPopover position="bottom right">
 						{ translate(
 							'The Site Icon is used as a browser and app icon for your site.' +
-								' Icons must be square, and at least %s pixels wide and tall.',
-							{ args: [ 512 ] }
+								' Icons must be square, and at least %s pixels wide and tall.' +
+								' {{a}}Learn more{{/a}}',
+							{
+								args: [ 512 ],
+								components: {
+									a: <InlineSupportLink supportContext="site-icons" />,
```

Comment:
Nit: not sure whether there's a hard and fast rule about this, but the rest of the Settings seem to have the `showIcon` prop set to false, so I'd say that it might be worth doing that here too. 

```suggestion
									a: <InlineSupportLink supportContext="site-icons" showIcon={ false } />,
```